force-app/main/default/lwc/brokerCard/__tests__/brokerCard.test.js

```diff
@@ -0,0 +1,164 @@
+const getRecordAdapter = registerLdsTestWireAdapter(getRecord);
+
+const BROKER_ID = 'a003h000003xlBiAAI';
+
+const BROKER_FIELDS_INPUT = [
+    {
+        fieldApiName: 'Name',
+        objectApiName: 'Broker__c'
+    },
+    {
+        fieldApiName: 'Phone__c',
+        objectApiName: 'Broker__c'
+    },
+    {
+        fieldApiName: 'Mobile_Phone__c',
+        objectApiName: 'Broker__c'
+    },
+    {
+        fieldApiName: 'Email__c',
+        objectApiName: 'Broker__c'
+    }
+];
+
+const OBJECT_API_NAME_INPUT = 'Broker__c';
+
+describe('c-broker-card', () => {
+    afterEach(() => {
+        // The jsdom instance is shared across test cases in a single file so reset the DOM
+        while (document.body.firstChild) {
+            document.body.removeChild(document.body.firstChild);
+        }
+
+        // Prevent data saved on mocks from leaking between tests
+        jest.clearAllMocks();
+    });
+
+    // Helper function to wait until the microtask queue is empty. This is needed for promise
+    // timing.
+    function flushPromises() {
+        // eslint-disable-next-line no-undef
+        return new Promise((resolve) => setImmediate(resolve));
+    }
+
+    describe('broker record form', () => {
+        it('gets property data from wire service', () => {
+            // Create element
+            const element = createElement('c-broker-card', {
+                is: BrokerCard
+            });
+            document.body.appendChild(element);
+
+            // Emit data from @wire
+            getRecordAdapter.emit(mockGetPropertyRecord);
+
+            return flushPromises().then(() => {
+                const propertyEl = element.shadowRoot.querySelector(
+                    'lightning-record-form'
+                );
+                expect(getFieldValue.mock.calls.length).toBe(1);
+                expect(propertyEl.recordId).toBe(BROKER_ID);
+            });
+        });
+
+        it('renders lightning-record-form with given input values', () => {
+            // Create element
+            const element = createElement('c-broker-card', {
+                is: BrokerCard
+            });
+            document.body.appendChild(element);
+
+            // Emit data from @wire
+            getRecordAdapter.emit(mockGetPropertyRecord);
+
+            // Return a promise to wait for any asynchronous DOM updates. Jest
+            // will automatically wait for the Promise chain to complete before
+            // ending the test and fail the test if the promise rejects.
+            return flushPromises().then(() => {
+                const propertyEl = element.shadowRoot.querySelector(
+                    'lightning-record-form'
+                );
+                expect(propertyEl.fields).toEqual(BROKER_FIELDS_INPUT);
+                expect(propertyEl.recordId).toBe(BROKER_ID);
+                expect(propertyEl.objectApiName).toBe(OBJECT_API_NAME_INPUT);
```

Comment:
No need to check the `objectApiName` attribute, it's static in the template.

Reply:
made the changes as suggested